app/javascript/src/components/Dashboard/Contacts/Table/ColumnData.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Avatar, Typography, Dropdown } from "neetoui";
+
+export const CONTACT_DETAILS_COLUMN_DATA = showDeleteAlert => [
```

Comment:
The file name and the data exported from this file doesn't resonate. Also go through the Slack thread.

Refer: https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react